CHANGELOG.en-US.md

```diff
@@ -15,6 +15,53 @@ timeline: true
 
 ---
 
+## 3.7.0
+3.7.0 is a heavy update that brings a lot of exciting changes and new features.
+- 🌟 Add new component : `Drawer`, demo it's [here](https://ant.design/components/drawer-cn/). [#10791](https://github.com/ant-design/ant-design/pull/10791)
+- 🌟 horizontal menu automatically collapses when there is no enough space.[#11234](https://github.com/ant-design/ant-design/pull/11234)
+![demo](https://user-images.githubusercontent.com/1731837/42550555-0fef7878-8505-11e8-9d2f-f708fec66b44.gif
+)
+- Upgrade `rc-tree-select` to `2.0.5` for `TreeSelect`, Refactored to fix the logic of the check.
+	- 🌟 Add `autoClearSearchValue` prop to clear the value of search input when multiple select is selected or deselected. [10996](https://github.com/ant-design/ant-design/issues/10996)
+	- 🌟 Add `searchValue` prop to set the value of search input. [6ff7dd8](https://github.com/ant-design/ant-design/commit/6ff7dd8fb953f079ee51ee638aaf4d832d0e10bf#diff-1e8e47abbdbe6e12d009aa61619ab22f)
+	- 🌟 Add `maxTagCount` prop to set the max count of visible tags. [fb96c9d](https://github.com/ant-design/ant-design/commit/fb96c9db351e44a202f64f780470c6319a8a9626)
+	- 🌟 Add `maxTagPlaceholder` prop to set the content when the tag is hidden. [fb96c9d](https://github.com/ant-design/ant-design/commit/fb96c9db351e44a202f64f780470c6319a8a9626)
+	- 🌟 Support to be case-sensitive for search input. [#10990](https://github.com/ant-design/ant-design/issues/10990)
+	- 🗑 Remove `label` prop and use `title` prop instead in the `treeData`.
+- Upgrade `rc-upload` to `2.5.0` for `Upload`. 
+	- 🌟 Add `directory` prop, support to upload directory. [#7315](https://github.com/ant-design/ant-design/issues/7315)
+	- 🌟 `action` prop supports to be the function which returns a `Promise` object. [fd96967](https://github.com/ant-design/ant-design/commit/fd96967c872600b79bb608e9ddf9f8c38814a704)
```

Comment:
`action` prop supports to be ~~the~~ a function which returns a `Promise` object.